textdb/textdb-exp/src/test/java/edu/uci/ics/textdb/exp/nlp/splitter/NlpSplitTest.java

```diff
@@ -0,0 +1,34 @@
+package edu.uci.ics.textdb.exp.nlp.splitter;
+
+import java.util.Arrays;
+import java.util.List;
+
+import org.junit.Test;
+
+import edu.uci.ics.textdb.api.exception.TextDBException;
+import edu.uci.ics.textdb.api.tuple.Tuple;
+import edu.uci.ics.textdb.exp.sink.tuple.TupleSink;
+import edu.uci.ics.textdb.exp.source.tuple.TupleSourceOperator;
+
+public class NlpSplitTest {
+
+    @Test
+    public void test1() throws TextDBException {
```

Comment:
One test case is not good enough. At least you need to have one test case for each split type (oneToOne and oneToMany).